samples/local-experiments.md

```diff
@@ -1,1079 +1,1150 @@
+```
 
+```json
+{
+  "outputs": [
+    {
+      "name": "OUTPUT",
+      "datatype": "FP32",
+      "shape": [
+        "4"
+      ],
+      "contents": {
+        "fp32Contents": [
+          10,
+          20,
+          30,
+          40
+        ]
+      }
+    }
+  ]
+}
 
-```python
-!cat ./experiments/addmul10.yaml
 ```
 
-    apiVersion: mlops.seldon.io/v1alpha1
-    kind: Experiment
-    metadata:
-      name: addmul10
-    spec:
-      default: pipeline-add10
-      resourceType: pipeline
-      candidates:
-      - name: pipeline-add10
-        weight: 50
-      - name: pipeline-mul10
-        weight: 50
-
+```bash
+cat ./experiments/addmul10.yaml
+```
 
+```yaml
+apiVersion: mlops.seldon.io/v1alpha1
+kind: Experiment
+metadata:
+  name: addmul10
+spec:
+  default: pipeline-add10
+  resourceType: pipeline
+  candidates:
+  - name: pipeline-add10
+    weight: 50
+  - name: pipeline-mul10
+    weight: 50
 
-```python
-!seldon experiment start -f ./experiments/addmul10.yaml 
 ```
 
-    {}
-
+```bash
+seldon experiment start -f ./experiments/addmul10.yaml
+```
 
+```json
+{}
 
-```python
-!seldon experiment status addmul10 -w | jq -M .
 ```
 
-    {
-      "experimentName": "addmul10",
-      "active": true,
-      "candidatesReady": true,
-      "mirrorReady": true,
-      "statusDescription": "experiment active",
-      "kubernetesMeta": {}
-    }
+```bash
+seldon experiment status addmul10 -w | jq -M .
+```
 
+```json
+{
+  "experimentName": "addmul10",
+  "active": true,
+  "candidatesReady": true,
+  "mirrorReady": true,
+  "statusDescription": "experiment active",
+  "kubernetesMeta": {}
+}
 
+```
 
-```python
-!seldon pipeline infer pipeline-add10 -i 50 --inference-mode grpc \
- '{"model_name":"add10","inputs":[{"name":"INPUT","contents":{"fp32_contents":[1,2,3,4]},"datatype":"FP32","shape":[4]}]}' 
+```bash
+seldon pipeline infer pipeline-add10 -i 50 --inference-mode grpc \
+ '{"model_name":"add10","inputs":[{"name":"INPUT","contents":{"fp32_contents":[1,2,3,4]},"datatype":"FP32","shape":[4]}]}'
 ```
 
-    Success: map[:add10_1::34 :mul10_1::16 :pipeline-add10.pipeline::34 :pipeline-mul10.pipeline::16]
+```
+Success: map[:add10_1::34 :mul10_1::16 :pipeline-add10.pipeline::34 :pipeline-mul10.pipeline::16]
 
+```
 
 Use sticky session key passed by last infer request to ensure same route is taken each time.
 
+```bash
+seldon pipeline infer pipeline-add10 --show-headers --inference-mode grpc \
+ '{"model_name":"add10","inputs":[{"name":"INPUT","contents":{"fp32_contents":[1,2,3,4]},"datatype":"FP32","shape":[4]}]}'
+```
 
-```python
-!seldon pipeline infer pipeline-add10 --show-headers --inference-mode grpc \
- '{"model_name":"add10","inputs":[{"name":"INPUT","contents":{"fp32_contents":[1,2,3,4]},"datatype":"FP32","shape":[4]}]}' 
 ```
+> /inference.GRPCInferenceService/ModelInfer HTTP/2
+> Host: 0.0.0.0:9000
+> seldon-model:[pipeline-add10.pipeline]
 
-    > /inference.GRPCInferenceService/ModelInfer HTTP/2

```

Comment:
:information_source: We're definitively improving over the existing solution here, where cURL formatting is erroneously interpreted as a quotation (for outbound metadata)